main.py

```diff
@@ -1,10 +1,49 @@
+import json
+
 import init_django_orm  # noqa: F401
 
 from db.models import Race, Skill, Player, Guild
 
 
 def main() -> None:
-    pass
+    with open("players.json", "r") as file:
+        players = json.load(file)
+
+    for player, player_data in players.items():
+        player_race = player_data["race"]
+        player_skills = player_race["skills"]
+        player_guild = player_data["guild"]
+
+        race, _ = Race.objects.get_or_create(
+            name=player_race["name"],
+            defaults={"description": player_race["description"]}
+        )
+
+        for skill_data in player_skills:
+            skill, _ = Skill.objects.get_or_create(
```

Comment:
btw you don't need to save object to the variable
```suggestion
            Skill.objects.get_or_create(
```